api/src/main/java/org/apache/cloudstack/api/command/user/template/RegisterTemplateCmd.java

```diff
@@ -162,6 +162,11 @@
                 description = "true if template should bypass Secondary Storage and be downloaded to Primary Storage on deployment")
     private Boolean directDownload;
 
+    @Parameter(name=ApiConstants.DEPLOY_AS_IS,
+            type = CommandType.BOOLEAN,
+            description = "(Vmware only) true if VM deployments should preserve all the configurations defined for this template")
```

Comment:
Can you please add "since" attribute here.

Reply:
```suggestion
            description = "(VMware only) true if VM deployments should preserve all the configurations defined for this template", since = "4.15.1")
```